src/components/Dropdown/Dropdown.tsx

```diff
@@ -2,69 +2,134 @@
 
 import { useOutsideClick } from '@/hooks/useOutsideClick';
 import classNames from 'classnames/bind';
-import { InputHTMLAttributes, forwardRef, useRef, useState } from 'react';
+import { InputHTMLAttributes, MouseEvent, forwardRef, useEffect, useRef, useState } from 'react';
+import TextField from '../TextField/TextField';
 import { Input, SuffixIcon } from '../parts';
 import styles from './Dropdown.module.scss';
 
 const cn = classNames.bind(styles);
 
-interface DropdownProps extends Omit<InputHTMLAttributes<HTMLInputElement>, 'size'> {
+interface DropdownProps extends InputHTMLAttributes<HTMLInputElement> {
   options: string[];
-  size?: 'sm' | 'md' | 'lg';
+  sizeVariant?: 'xs' | 'sm' | 'md' | 'lg';
+  className?: string;
+  onClick?: () => void;
 }
 
 /**
  * Dropdown component documentation
  *
  * @param {object} props - 컴포넌트의 속성
  * @param {string} [props.type='text'] - 입력 필드의 타입, 기본값은 'text'
- * @param {string} [props.size='sm'] - 드롭다운의 크기, 'sm', 'md', 'lg' 중 하나
+ * @param {string} [props.sizeVariant='sm'] - 드롭다운의 사이즈:
+ * 'xs' = sort Dropdown에서 사용,
+ * 'sm' = 일반적인 상품 옵션 선택,
+ * 'md', 'lg' 중 하나
  * @param {string[]} props.options - 선택할 수 있는 옵션 목록
  * @param {string} [props.placeholder] - placeholder 텍스트로 기본값 설정
+ * placeholder값 선택시 inputValue === ('')
+ * placeholder 설정하지 않을 시 options의 첫번째 값 렌더링
  * @returns {JSX.Element} 렌더링된 드롭다운 컴포넌트
  */
 
 export default forwardRef<HTMLInputElement, DropdownProps>(function Dropdown(
-  { type = 'text', size = 'sm', options, ...rest },
+  { type = 'text', sizeVariant = 'sm', options, className, onClick, ...rest },
   ref,
 ) {
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const [dropdownValue, setDropdownValue] = useState<string>('');
+  const [isTextFieldVisible, setIsTextFieldVisible] = useState(false);
   const DropdownRef = useRef<HTMLDivElement>(null);
+  const TextareaRef = useRef<HTMLTextAreaElement>(null);
+
+  useEffect(() => {
+    if (!rest.placeholder) {
+      setDropdownValue(options[0]);
+    }
+  }, [options, rest.placeholder]);
 
   useOutsideClick(DropdownRef, () => {
+    if (TextareaRef.current) {
```

Comment:
의도하신 건진 모르겠지만 직접 입력하게 했을 때 빈 값은 안 받는 게 좋지 않을까요?
```suggestion
    if (TextareaRef.current?.value) {
```

Reply:
오오 이런 디테일 좋습니다 제가 놓쳤군요